src/containers/SaveTemplateButton/__snapshots__/SaveTemplateButton.test.jsx.snap

```diff
@@ -12,10 +12,10 @@ exports[`<SaveTemplateButton /> renders correctly in disabled state 1`] = `
   <span
     className="d-flex align-items-center justify-content-center"
   >
-    <span
-      className=""
-    >
+    <span>
+       
       Save Template
+       
```

Comment:
As noted in another comment, if you upgrade to latest Paragon (16.14.7), this snapshot will be a bit different (without the leading/trailing spaces).